crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1296,6 +1296,11 @@ impl
                 }))
             }
             api::PaymentMethodData::Wallet(_) => Ok(Self::Wallet),
+            _ => Err(errors::ConnectorError::NotSupported {
```

Comment:
Avoid using underscore to handle other cases, using underscore we can find NotSupported only in runtime and not in compile time.